packages/checkbox/index.js

```diff
@@ -69,7 +69,9 @@ export class Checkbox extends Component {
   }
 
   componentWillUnmount() {
-    this.foundation_.destroy();
+    if (this.foundation_) {
+      this.foundation_.destroy();
+    }
```

Comment:
This change (and all the others like it below) are not testable.
This means the PR here will bring the coverage amount down substantially.

It's a bit of a chicken/egg scenario as I need the foundation **to not exist** to check that `destroy()` is not called, but I need the foundation **to exist** to stub out `destroy()`. Sadface.

Reply:
> Merging #512 into master will increase coverage by 0.01%

Huh? If you say so _codecov_.